addons/portal/views/portal_templates.xml

```diff
@@ -260,7 +260,7 @@
                         <span t-if="placeholder_count" t-att-class="'' if show_count else 'd-none'" t-att-data-placeholder_count="placeholder_count"/>
                         <span t-out="title"/>
                     </h5>
-                    <p class="m-0 text-600">
+                    <p class="m-0 opacity-50">
```

Comment:
Note: I'm not sure this is the right approach, it seems weird to have to apply an opacity on a text to ensure it is readable. Perhaps something to handle with  --o-cc variables (?)